spec/system/referrals/user_adds_contact_details_email_spec.rb

```diff
@@ -0,0 +1,101 @@
+# frozen_string_literal: true
+require "rails_helper"
+
+RSpec.feature "User adds the contact details email to a referral" do
+  scenario "Views referral summary sections" do
```

Comment:
```suggestion
RSpec.feature "Contact details" do
  scenario "User submits contact details for the referred person" do
```

- with an update to the filename to match the scenario
- and then this spec could be used for all further contact detail steps in the flow

Otherwise spec structure below looks 👌🏼